frontend/lib/service-definitions/index.ts

```diff
@@ -7,20 +7,24 @@ export interface Category {
   name: string
 }
 
+export interface ExampleSignature {
+  pattern: string
+  namedParams: { [k: string]: string }
+  queryParams: { [k: string]: string }
+}
+
+export interface Preview {
+  label?: string
```

Comment:
Could you remind me again why label is optional (feel like I remember asking this before sorry 😄)? Is it because it's often set in the default badge data for a service or a different reason?

Reply:
I think this was to match the schema of the service definition export. If that always returns an empty string when there is no label, we could update this.

Reply:
Gotcha thanks